iceoryx_binding_c/include/iceoryx_binding_c/service_description.h

```diff
@@ -17,13 +17,15 @@
 #ifndef IOX_BINDING_C_SERVICE_DESCRIPTION_H
 #define IOX_BINDING_C_SERVICE_DESCRIPTION_H
 
+#include "iceoryx_binding_c/config.h"
+
 #include <stdint.h>
 
 typedef struct
 {
-    char serviceString[100U];
-    char instanceString[100U];
-    char eventString[100U];
+    char serviceString[IOX_CONFIG_SERVICE_STRING_SIZE + 1U]; // +1U for \0 terminator
```

Comment:
Please do not do this. `IOX_CONFIG_SERVICE_STRING_SIZE` should contain the actual capacity of the string including the null terminator.
Otherwise we have a weird behavior when for instance we have `sizeof` calculations etc.

Reply:
I would argue the string size is n and the buffer size is n +1.